crates/typst/src/visualize/path.rs

```diff
@@ -33,11 +33,13 @@ pub struct PathElem {
     ///
     /// When setting a fill, the default stroke disappears. To create a
     /// rectangle with both fill and stroke, you have to configure both.
-    ///
-    /// Currently all paths are filled according to the [non-zero winding
-    /// rule](https://en.wikipedia.org/wiki/Nonzero-rule).
     pub fill: Option<Paint>,
 
+    /// The rule used to fill the path.
+    /// Defaults to `non-zero`.
+    #[default(FillRule::default())]
+    pub fill_rule: FillRule,
```

Comment:
As `FillRule` implements the `Default` trait, I think this can be shortened to just `#[default]`. Also, the note about the default value can be removed from the comment as it is added automatically to the docs with the attribute.